host/class/cdc/esp_modem_usb_dte/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 idf_component_register(SRCS "esp_modem_usb.cpp" "esp_modem_usb_api_target.cpp" "esp_modem_usb_c_api.cpp"
-                       REQUIRED_IDF_TARGETS esp32s2 esp32s3
+                       REQUIRED_IDF_TARGETS esp32s2 esp32s3 esp32p4
```

Comment:
This line can be removed. The required target is now taken from https://github.com/espressif/esp-usb/blob/master/host/class/cdc/usb_host_cdc_acm/idf_component.yml#L7